### ISSUE TITLE
Question about quality

### ISSUE THREAD

**Issue by ClearSky10** [kan-bayashi/ParallelWaveGAN#199]
Hello.

We are trying to synthesize female English voice using PWG with multi band melgan config. We've trained PWG on ~20 hours of data.
However, we were unable to achieve solid results.

[Here's notebook with synth examples](https://colab.research.google.com/drive/16c26Bz_9BLeLkWSvd6la4mNWipyA4UNX)
[Here's notebook with our Tensorboard logs and original logs](https://colab.research.google.com/drive/1uWFFIDLk_l9toLZlIPTA9bnxDh4f4YOY)
[Here's notebook with synth examples with official checkpoint for our config](https://colab.research.google.com/drive/1n6pfdZrJEKHYrLyKwTVHQ9oyfDwcTykB)

We can give additional info if needed.

Could you please provide insights on this situation? Do we have bad data? Perhaps we are using inappropriate config?

**Comment by kan-bayashi** [comment]
Sorry for late replay.
Let me confirm some points:
- Could you explain the difference of the two notebook? You mean your trained model is not comparable to my pretrained model?
- Please provide your configuration and dataset detail (e.g. sampling rate).

**Comment by ClearSky10** [comment]
Yes, there's difference in quality. Our model is not as good as yours.
We've used multi band melgan v1 from ljspeech, sampling rate is 22050.

**Comment by kan-bayashi** [comment]
Please try v2 config, which will give better results.
Also, it is better to remove the silence as much as possible.

**Comment by Alexey322** [comment]
@kan-bayashi do you mean silence at the beginning and at the end of the audio? Or the length of the pauses between the pronounced words should also be minimized?

**Comment by kan-bayashi** [comment]
> do you mean silence at the beginning and at the end of the audio?

Basically, yes. But if the audio contains long silence (say, over 1 seconds) in the middle, it is better to consider shorten the silence.

**Comment by Alexey322** [comment]
@kan-bayashi, Can you please explain why silence affects quality?
I can remove all the silence completely with librosa.effects.split, but will this affect the quality during synthesis? In this case, mel spectrograms will be synthesized along with silence.

**Comment by kan-bayashi** [comment]
The discriminator cannot distinguish the silence part. Therefore if the silence part is dominant in the batch, the discriminator loss will be high even if the voiced part is not so good. This causes performance degradation especially when we use the small amount of the training data such as adaptation case.
